sass/_header.scss

```diff
@@ -5,7 +5,7 @@
   top: 0;
   left: 0;
   right: 0;
-  height: $header-height;
+  //height: $header-height;
```

Comment:
.

Reply:
you can take care of this in the follow up PR when introducing back the dark-mode